README.md

```diff
@@ -52,6 +52,12 @@ To have it up and running please execute the following steps:
 [Download and install Ollama](https://ollama.com/download)
 [Download and install Docker](https://www.docker.com/products/docker-desktop/)
 
+(MAC) 
+```
+brew install poppler
+brew install libmagic
```

Comment:
Please move it to a `Manual setup` section, because it's already included in makefile
 https://github.com/CatchTheTornado/text-extract-api/blob/afe9eeaddf4a7807f5ca65d78b9328a3287bc99a/Makefile#L74

Reply:
done